apps/docs/src/.vitepress/config.ts

```diff
@@ -29,62 +29,31 @@ export default defineConfig({
       { text: "Q&A", link: "https://github.com/schwarzit/onyx/discussions/categories/q-a" },
     ],
     socialLinks: [{ icon: "github", link: packageJson.repository.url }],
-    sidebar: {
-      // default sidebar items
-      "/": [
-        {
-          text: "Introduction",
-          collapsed: false,
-          items: [
-            { text: "Getting Started", link: "/getting-started" },
-            { text: "The Team", link: "/team" },
-          ],
-        },
-        {
-          text: "Advanced",
-          collapsed: false,
-          items: [
-            {
-              text: "Additional packages",
-              link: "/packages/",
-            },
-          ],
-        },
-        {
-          text: "Components",
-          base: "/components",
-          collapsed: false,
-          items: getComponents().map((name) => ({ text: name, link: `/${name}` })),
-        },
-      ],
-      // standalone sidebar for documentation about our additional packages
-      "/packages/": [
-        {
-          text: "Introduction",
-          items: [
-            { text: "Back to Onyx", link: "/getting-started" },
-            { text: "About", link: "/packages/" },
-          ],
-        },
-        {
-          text: "Additional packages",
-          base: "/packages",
-          items: [
-            {
-              text: "@sit-onyx/figma-utils",
-              link: "/figma-utils",
-            },
-            {
-              text: "@sit-onyx/headless",
-              link: "/headless",
-            },
-            {
-              text: "@sit-onyx/storybook-utils",
-              link: "/storybook-utils",
-            },
-          ],
-        },
-      ],
-    },
+    sidebar: [
+      {
+        text: "Introduction",
+        collapsed: false,
+        items: [
+          { text: "Getting Started", link: "/getting-started" },
+          { text: "The Team", link: "/team" },
+        ],
+      },
+      {
+        text: "Components",
+        base: "/components",
+        collapsed: false,
+        items: getComponents().map((name) => ({ text: name, link: `/${name}` })),
+      },
+      {
+        text: "Other npm packages",
```

Comment:
```suggestion
        text: "@sit-onyx npm packages",
```
or
```suggestion
        text: "@sit-onyx packages",
```
or
```suggestion
        text: "Other onyx packages",
```
maybe? :)

Reply:
I did a combination of both and changed it to "Other Onyx npm packages"^^